src/main/java/core/basesyntax/FileWork.java

```diff
@@ -1,7 +1,45 @@
 package core.basesyntax;
 
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.List;
+
 public class FileWork {
+    public static final char FILTER_CHARACTER = 'w';
+    public static final String SEPARATOR_CHARACTER = "_";
+    public static final String SECOND_SEPARATOR_CHARACTER = " ";
+    public static final String EMPTY_CHARACTER = "";
+
     public String[] readFromFile(String fileName) {
-        return null;
+        try {
+            String res = "";
+            String list = "";
```

Comment:
let's use String Builder instead of `String list` here
it is better to move this variable where we use it
the sane with `String res`